pages/settings/index.js

```diff
@@ -861,10 +861,28 @@ function AuthMethods ({ methods, apiKeyEnabled }) {
             : <div key={provider} className='mt-2'><EmailLinkForm callbackUrl='/settings' /></div>
         } else if (provider === 'lightning') {
           return (
+            <div key={provider} className="d-flex align-items-center">
             <QRLinkButton
               key={provider} provider={provider}
               status={methods[provider]} unlink={async () => await unlink(provider)}
             />
+            <Info className='mt-2'>
+                      <ul>
+                        <li>
+                          This is an LNURL-auth. Not sure what it is? Check <a target="_blank" href="https://lightninglogin.live/learn">here</a>
+                        </li>
+                        <li>
+                          Linking your Lightning wallet allows you to quickly sign in and zap directly from your wallet.
```

Comment:
> zap directly from your wallet.

https://github.com/stackernews/stacker.news/pull/2206#pullrequestreview-2898198255 ...

I will not review your code again for at least a week because you write QA 10 and then just repeat mistakes we already told you about.

This makes me believe you did not review or test your own code at all.